bin/provisioner.js

```diff
@@ -1,7 +1,7 @@
 let debug = require('debug')('aws-provisioner:bin:provisioner');
 let base = require('taskcluster-base');
 let provision = require('../lib/provision');
-let Aws = require('multi-region-promised-aws');
+let awsSdk = require('aws-sdk-promise');
```

Comment:
nit, but everywhere else we do: `let aws = require('aws-sdk-promise');`
Just `aws` as variable...